docs/DeveloperGuide.md

```diff
@@ -532,9 +532,15 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is `TAlent Assistant™ Program` and the **Actor** is the `professor`, unless specified otherwise)
+(For all use cases below, the **System** is `TAlent Assistant™` and the **Actor** is the `professor`, unless specified otherwise)
 
-> Note: TA - Teaching Assistant
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes on the abbreviations:**<br>
+
+TA - Teaching Assistant
```

Comment:
I remember you brought up this issue of standardizing whether to use TA or candidates, perhaps once we finalize it this might need to change as well.

Reply:
Thanks for reviewing En En! Yup we gotta decide on it. I think should be candidates as this program helps to manage candidates who are applying to become TAs.